ndk-build/src/manifest.rs

```diff
@@ -70,6 +70,9 @@ pub struct Application {
     #[serde(rename(serialize = "android:label"))]
     #[serde(default)]
     pub label: String,
+    #[serde(rename(serialize = "android:name"))]
+    #[serde(default)]
+    pub name: String,
```

Comment:
https://developer.android.com/guide/topics/manifest/application-element

> `android:name`
>
> The fully qualified name of an `Application` subclass implemented for the application. When the application process is started, this class is instantiated before any of the application's components.
>
> The subclass is optional; most applications won't need one. In the absence of a subclass, Android uses an instance of the base Application class.

I don't think that's better. IMO we should add a separate `apk_name` field to:

https://github.com/rust-windowing/android-ndk-rs/blob/7936944edc699d3e7f380cfa87515f8899ce7027/cargo-apk/src/manifest.rs#L7-L13

And fall back to the artifact name as usual.